lib/internal/per_context/domexception.js

```diff
@@ -37,14 +42,36 @@ function ensureInitialized() {
   isInitialized = true;
 }
 
+// This is the same as https://github.com/nodejs/node/blob/12e3c74e2edf78eeae662dce0a9db17a6fe7d730/lib/internal/worker/js_transferable.js#L41-L46.
+// It has been copied here because require() is not available inside this
+// context.
+function makeTransferable(obj) {
+  const inst = ReflectConstruct(JSTransferable, [], obj.constructor);
+  ObjectDefineProperties(inst, ObjectGetOwnPropertyDescriptors(obj));
+  ObjectSetPrototypeOf(inst, ObjectGetPrototypeOf(obj));
+  return inst;
+}
+
 class DOMException extends Error {
   constructor(message = '', name = 'Error') {
     ensureInitialized();
     super();
-    internalsMap.set(this, {
-      message: `${message}`,
-      name: `${name}`
+    const code = nameToCodeMap.get(name) || 0;
+    // We need to define these properties on this throw-away object because
+    // makeTransferable() indirectly invokes Error#toString(), which calls the
+    // getters on `this` object but the getters are designed to work only on the
+    // `internals` object.
+    ObjectDefineProperties(this, {
+      message: { value: message },
```

Comment:
You probably want to define explicitly `enumerable`, `configurable`, and `writable` in case their values has been overwritten on `Object.prototype`.

Reply:
@aduh95 thanks, done in https://github.com/nodejs/node/pull/41148/commits/6bfbc161199cb2087d3cb8a45f6f05e198b96374